### ISSUE TITLE
creating passwords

### ISSUE THREAD

**Issue by LeanoJ** [Callidus2000/SecretManagement.NetwrixPasswordSecure#3]
Is it already known whether the feature for creating passwords will be implemented?
It would also be possible to specify the owner and OU and a rights template.

**Comment by Callidus2000** [comment]
First of all: Sorry for the late (and incomplete) response. If anyone could give me a hint where to buy free time in large bottles I would be happy (as my work and the necessary private activities don't leave available).

A lot of time has passed since I created the module, unfortunately it has not been used very often. 

If I'm not mistaken, then I should have realized adding it already, what surprises me is the contrary statement in the readme file. However, my colleague has apparently reinstalled the integration instance of the server. So I currently have no way of checking whether the readme or my memory is correct. I have told him that I need access again and will then take a look at the current status.

Sorry

**Comment by LeanoJ** [comment]
To be honest, it didn't work for me to create passwords. However, it is of course possible that I didn't do something right.

Since I am currently writing a Powershell script (I'm not very good though) that creates VMs and can create local administrator passwords directly, it would of course be super handy to automatically integrate them into Passwordsecure.

A short info or the provision of a newer version would certainly be super helpful! I would be very thankful!

I apologise for the late reply. I'm not that active here.

**Comment by Callidus2000** [comment]
No need to apologise. I hope my colleague provides me with the server tomorrow. Otherwise I'll be back in office in 4 weeks.

**Comment by LeanoJ** [comment]
Hey, I didn't mean to be annoying. Just wanted to ask if anything had already come up :)

**Comment by Callidus2000** [comment]
Yes, responsibility has shifted towards a very young colleague. Further help in this issue's direction? Sadly no.

BTW: Today we nearly lost access to all our passwords....
